db/seeds.rb

```diff
@@ -9,3 +9,8 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the
 # Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+admin = Admin.create!(
```

Comment:
it'l rise error if you run it more then once or you've got already record with this `email`. But we don't  need it. 
Please return to `create` instead of  `create!`, or use `find_or_create`